### ISSUE TITLE
Docs missing for ansi.Cursor?

### ISSUE THREAD

**Issue by Evidlo** [tartley/colorama#246]
The README mentions moving the cursor around in `demo06.py`, but this is with manually generated ANSI codes and doesn't make use of `ansi.Cursor`.

I only realized `ansi.Cursor` existed because it was mentioned in [this PR](https://github.com/tartley/colorama/pull/210).

**Comment by wiggin15** [comment]
You're right, the demo is a little outdated and we are missing documentation for `ansi.Cursor` in our README. If you have the time, we'll be happy to accept pull requests!
